tinyagentos/projects/project_store.py

```
@@ -67,25 +68,61 @@ class ProjectStore(BaseStore):
             "ALTER TABLE project_members ADD COLUMN can_edit_canvas INTEGER NOT NULL DEFAULT 0",
+            "ALTER TABLE project_members ADD COLUMN can_read_canvas INTEGER NOT NULL DEFAULT 0",
             "ALTER TABLE project_members ADD COLUMN is_lead INTEGER NOT NULL DEFAULT 0",
             "ALTER TABLE projects ADD COLUMN user_id TEXT NOT NULL DEFAULT ''",
+            "ALTER TABLE projects ADD COLUMN lead_member_id TEXT",
         ):
             try:
                 await self._db.execute(col_def)
                 await self._db.commit()
             except Exception:
                 # Column already exists on fresh installs (created by SCHEMA).
                 pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)